src/wire-protocol.ts

```diff
@@ -73,8 +73,8 @@ export type QueryStats = {
 };
 
 export type QueryInfo = {
-  /** The last transaction time of the query. An ISO-8601 date string. */
-  txn_time: string;
+  /** The last transaction timestamp of the query. An Unix epoch in microseconds. */
```

Comment:
Nit: I've never seen an before unix.

```suggestion
  /** The last transaction timestamp of the query. A Unix epoch in microseconds. */
```